harwest/lib/utils/submissions.py

```diff
@@ -34,11 +34,17 @@ def __generate_readme(self, submissions):
         continue
       problems.add(submission['problem_url'])
       row = str(index) + " | "
-      row += '[{problem_index} - {problem_name}]({problem_url}) | '.format(
-        problem_index=submission['problem_index'],
-        problem_name=submission['problem_name'],
-        problem_url=submission['problem_url']
-      )
+      if submission['platform'] == 'atcoder':
```

Comment:
Users who might already have used harwest before to set up their repository will have the `platform` key missing in their submissions.js in which case it'll throw a KeyError. Let's just have a safe check of `'platform' in submission` before we compare it to atcoder. For cases wherein the key is missing, we can safely assume it to be from codeforces

Reply:
yeah, that'll be better